proposals/0002-root-signature-in-clang.md

```diff
@@ -355,6 +371,8 @@ checks are concerned.
 
 The root signature AST nodes are serialized / deserialized as normal bitcode.
 
+The root signature version must be added as one of the parameters for the in-memory datastructures.
+
```

Comment:
```suggestion
```

This proposal is defining the in-memory data structures.  Although this PR is adding a version requirement, the proposal itself shouldn't say that we're adding something to data structures because this proposal is defining what they are.